runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -73,6 +73,7 @@
 #define OPT_NUMA_NONE "-Xnuma:none"
 #define OPT_XXMAXRAMPERCENT "-XX:MaxRAMPercentage="
 #define OPT_XXINITIALRAMPERCENT "-XX:InitialRAMPercentage="
+#define OPT_XCMCR "-Xcmcr"
```

Comment:
the option should go to mmparsexxgc.cpp (fine tuning options, mostly for developers and debugging, that we normally don't expose to customers) and not be an acronym.